sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/codegen/BufferHolderSparkSubmitSuite.scala

```diff
@@ -39,8 +39,8 @@ class BufferHolderSparkSubmitSuite
     val argsForSparkSubmit = Seq(
       "--class", BufferHolderSparkSubmitSuite.getClass.getName.stripSuffix("$"),
       "--name", "SPARK-22222",
-      "--master", "local-cluster[2,1,1024]",
-      "--driver-memory", "4g",
+      "--master", "local-cluster[1,1,7168]",
```

Comment:
Do we still support this?

Reply:
Good question. Several tests still seem to use `local-cluster`. Is it better to use `local` while it may require more memory?

Reply:
ping @hvanhovell

Reply:
I think we support this for debugging purpose since, IIRC, that's going to make separate processes for workers.